buildbot/master.cfg

```diff
@@ -443,6 +443,28 @@ def make_android_package(mode='normal'):
                       descriptionDone="build",
                       haltOnFailure=True))
 
+    f.addStep(ShellCommand(command=["mkdir", "-p", "build-arm64"],
+                           logEnviron=False,
+                           description="mkbuilddir",
+                           descriptionDone="mkbuilddir"))
+
+    f.addStep(ShellCommand(command=["cmake", "-DANDROID=True", "-DANDROID_NATIVE_API_LEVEL=android-18",
+                                    "-DGIT_EXECUTABLE=/usr/bin/git",
+                                    "-DANDROID_ABI=arm64-v8a",
+                                    "-DANDROID_TOOLCHAIN_NAME=aarch64-linux-android-4.9"
+                                    "-DCMAKE_TOOLCHAIN_FILE=../Source/Android/android.toolchain.cmake", ".."],
+                           env={ "ANDROID_NDK": "/home/buildbot/ndk" },
+                           workdir="build/build-arm64",
+                           description="configuring",
```

Comment:
Update steps description for both ARMv7 and your new ARMv8 in order to say what arch they're building for. For example, "configure armv7", "build armv8", ...